main.js

```diff
@@ -0,0 +1,66 @@
+for (let i = 10; i <= 25; i = i + 0.5) {
+  console.log(i);
+}
+
+
+const number = +prompt("Уведіть ваше число?");
+let remainder = 0;
+
+for (let i = 1; i <= 100; i++) {
+  if (number % i === 0) {
+    ++remainder;
+  }
+  if (i === 100) {
+    console.log(`Число ${number} ${remainder > 2 ? "не" : "є"} просте`);
+    console.log(remainder);
+  }
+}
+
+const discount = +prompt(`Ваша сума покупки ?`);
```

Comment:
As for me, vthe ariable name does not reflect value context, `discount` is about the discount itself, not about a user   purchase sum